lib/Inline/Perl5/ClassHOW.pm6

```diff
@@ -123,7 +123,7 @@ class Inline::Perl5::ClassHOW
 
     my $archetypes := Metamodel::Archetypes.new(
         :nominal(1), :inheritable(1), :augmentable(1) );
-    method archetypes() {
+    method archetypes($obj?) {
```

Comment:
If you're not using the parameter, then `$?` should be enough?

Reply:
Indeed. NQP inertia, as I was doing it all over the NQP code shortly before.